src/app/games/mancala/kalah/KalahTutorial.ts

```diff
@@ -0,0 +1,85 @@
+            MancalaState.getInitialState(),
+            [
+                '#click_0_1',
+                '#click_1_1',
+                '#click_2_1',
+            ],
+            $localize`As you see, three houses have been fed in addition to your kalah.`,
+            $localize`Failed. Choose the three lefter house on the bottom.`,
+        ),
+        TutorialStep.fromPredicate(
+            $localize`The Kalah (2/2)`,
+            $localize`When ending in the Kalah, you are allowed to distribute again.<br/><br/>You're playing Dark, play the house that end up in the Kalah then do a second move!`,
+            MancalaState.getInitialState(),
+            KalahMove.of(MancalaDistribution.THREE, [MancalaDistribution.ONE]),
+            (move: KalahMove, _previous: MancalaState, _result: MancalaState) => {
+                if (move.subMoves.length === 1) {
+                    return MGPValidation.failure($localize`This move only distributed one house, do one distribution that end in the Kalah then a second one!`);
+                } else {
+                    return MGPValidation.SUCCESS;
+                }
+            },
+            $localize`Congratulations!`,
+        ),
+        TutorialStep.fromPredicate(
+            $localize`Capture`,
+            $localize`When the last seed of a distribution end up in one of your empty houses, if the house over it is filled, then you capture both houses. On this board, such a move is possible.<br/><br/>You're playing Dark, do a capture!`,
+            new MancalaState([
+                [0, 4, 4, 4, 4, 4],
+                [0, 2, 0, 2, 4, 0],
+            ], 4, [0, 0]),
+            KalahMove.of(MancalaDistribution.ONE, [MancalaDistribution.ZERO, MancalaDistribution.THREE]),
+            (_move: KalahMove, _state: MancalaState, resultingState: MancalaState) => {
+                if (resultingState.getPieceAtXY(1, 0) === 0) {
+                    return MGPValidation.SUCCESS;
+                } else {
+                    return MGPValidation.failure('You did not capture, try again!');
+                }
+            },
+            $localize`Congratulations!`,
+        ),
+        TutorialStep.fromMove(
+            $localize`End of the game`,
+            $localize`At any moment, when one player have more than 24 seeds in their kalah, they win. That can happend before the board is emptied, but, there is also a second way. When, at the beginning of your turn, you don't have any seed in your house, the game is over and your opponent takes all the remaining seeds from his houses. Here, the opponent's just gave you their last seed, if you manage not to distribute any seeds in their houses, you win.<br/><br/>You're playing Dark, win!`,
```

Comment:
happen (sans d)
before the board is empty.
enlever "at the beginning of your turn"
in your house -> on your side
his  -> their
the opponent's -> your opponent (sans 's et avec un your)
arrêter la phrase avant le "if" et commencer une nouvelle